nin/frontend/app/scripts/utils/render.js

```diff
@@ -19,8 +18,13 @@ commands.on('stopRendering', () => {
 
 function render(i) {
   i = i || 0;
-  currentFrame = i;
   demo.jumpToFrame(i);
+  const currentFrame = demo.getCurrentFrame();
+  if (currentFrame < i) {
```

Comment:
perhaps this should be `currentFrame <= i` so that we can render the final frame of the demo as well?